Mathlib/Topology/Basic.lean

```diff
@@ -1640,15 +1641,24 @@ theorem preimage_interior_subset_interior_preimage {f : α → β} {s : Set β}
   interior_maximal (preimage_mono interior_subset) (isOpen_interior.preimage hf)
 #align preimage_interior_subset_interior_preimage preimage_interior_subset_interior_preimage
 
+@[continuity]
 theorem continuous_id : Continuous (id : α → α) :=
   continuous_def.2 fun _ => id
 #align continuous_id continuous_id
 
+@[continuity]
+theorem continuous_id' : Continuous (fun (x : α) => x) := continuous_id
```

Comment:
I would put this in `Mathlib/Tactic/Continuity` and/or add a note saying that this exists because of semireducible transparency issues with `continuous_id` and the continuity tactic.
```suggestion
theorem continuous_id' : Continuous (fun (x : α) => x) := continuous_id
```

Reply:
I can't put it in the tactic file, since the tactic actually knows nothing about topological spaces; the import structure got reversed.